JoelFaldin/ServerSort-MuniCrud-Backend
# Testing/departments service test

# Add tests for departments service

## Description 🗒️✏️

Added tests for departments.service, just like the others!

## How to test it? 🎯⚗️🔬

- Install the dependencies using `pnpm i`.
- For prisma, run `pnpx prisma migrate dev --name init`, then run `pnpx prisma generate`. Aditionally, you can start a local page to inspect the tables using `pnpx prisma studio`, but is not necessary for the tests.
- Run `pnpm run test`, and you should see something like this:
![image](https://github.com/user-attachments/assets/fba15485-6264-49f0-bbbe-d9f01b655740)
- Alternatively, you can run `pnpm run test -- src/departments/__tests__/` to run tests in the `directions` folder.
